ZipBuilder/Sources/ZipBuilder/CocoaPodUtils.swift

```diff
@@ -327,7 +327,15 @@ enum CocoaPodUtils {
                                                  in installedPods: [String: PodInfo])
     -> [VersionedPod] {
     return transitivePodDependencies(for: podName, in: installedPods).map {
-      CocoaPodUtils.VersionedPod(name: $0, version: installedPods[$0]?.version)
+      var podVersion: String?
+      if let version = installedPods[$0]?.version {
+        podVersion = version
+      } else {
+        // See if there's a version on the base pod.
+        let basePod = String($0.split(separator: "/")[0])
```

Comment:
Is it possible for this to out-of-index for dependencies that aren't subspecs, e.g. nanopb?

Reply:
My understanding of split is there will always be a 0th element for a non-nil string.